igor-web/src/main/java/com/netflix/spinnaker/igor/artifactory/model/ArtifactoryRepositoryType.java

```diff
@@ -17,5 +17,6 @@
 package com.netflix.spinnaker.igor.artifactory.model;
 
 public enum ArtifactoryRepositoryType {
-  Maven,
+  MAVEN,
```

Comment:
🤔 I believe this change is backwards-incompatible. I agree that enums should be capitalized, but if this enum has been serialized anywhere, we'll need to store the value as a string instead...

```java
public enum ArtifactoryRepositoryType {
  MAVEN("Maven"),
  HELM("HELM");

  // ...
}

Reply:
Thank you  @robzienert good point , I will address that.

Reply:
Hi @robzienert thank you for the review.

Reply:
> 🤔 I believe this change is backwards-incompatible. I agree that enums should be capitalized, but if this enum has been serialized anywhere, we'll need to store the value as a string instead...
> 
> ```java
> public enum ArtifactoryRepositoryType {
>   MAVEN("Maven"),
>   HELM("HELM");
> 
>   // ...
> }
> ```

Just so I understand , so the implementation will be 
`
public enum ArtifactoryRepositoryType {
    MAVEN("Maven"),
    HELM("HELM");
    String repoTypeString;
    ArtifactoryRepositoryType(String repoTypeString)
    {
       this.repoTypeString = repoTypeString;
    }
    public String getRepoTypeString() {
        return repoTypeString;
    }
}
`
To map the search type in ArtifactorySearch , it could still be 
`public ArtifactoryRepositoryType getRepositoryType() {
    return ArtifactoryRepositoryType.valueOf(repoType.toUpperCase());
  }`
The references to ArtifactoryRepositoryType in ArtifactoryBuildMonitor and ArtifactoryItem could use the enum or be updated to use the string value instead repoType.getRepoTypeString()

Would that be accurate?